.php_cs

```diff
@@ -1,37 +1,6 @@
 <?php
-$finder = Symfony\CS\Finder\DefaultFinder::create()
-    ->in('src');
-$config = Symfony\CS\Config\Config::create();
-$config->level(null);
-$config->fixers(
-    array(
-        'braces',
-        'duplicate_semicolon',
-        'elseif',
-        'empty_return',
-        'encoding',
-        'eof_ending',
-        'function_call_space',
-        'function_declaration',
-        'indentation',
-        'join_function',
-        'line_after_namespace',
-        'linefeed',
-        'lowercase_keywords',
-        'parenthesis',
-        'multiple_use',
-        'method_argument_space',
-        'object_operator',
-        'php_closing_tag',
-        'remove_lines_between_uses',
-        'short_array_syntax',
-        'short_tag',
-        'standardize_not_equal',
-        'trailing_spaces',
-        'unused_use',
-        'visibility',
-        'whitespacy_lines',
-    )
-);
-$config->finder($finder);
-return $config;
+$config = new Prooph\CS\Config\Prooph();
+$config->getFinder()->in(__DIR__ . '/src')->in(__DIR__ . '/tests');
+$cacheDir = getenv('TRAVIS') ? getenv('HOME') . '/.php-cs-fixer' : __DIR__;
```

Comment:
@prolic I have taken this from event-sourcing, how ever this directory "/home/travis/.php-cs-fixer" causes issues, any ideas?
https://travis-ci.org/prooph/laravel-package/jobs/230918730#L472